resources/assets/less/bem/forum-topic-title.less

```diff
@@ -0,0 +1,68 @@
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+.forum-topic-title {
+  // actual font takes more than the 1.1 default line height.
+  line-height: 1.5;
+
+  &__button {
+    margin-left: 5px;
+    display: flex;
+    pointer-events: auto;
+  }
+
+  &__buttons {
+    padding-left: 20px;
+    display: flex;
+  }
+
+  &__buttons-container {
+    position: absolute;
+    left: 0;
+    top: 0;
+    display: flex;
+    align-items: center;
+    height: 100%;
+    pointer-events: none;
+    width: 100%;
+  }
+
+  &__buttons-padding {
+    pointer-events: none;
+    opacity: 0;
+    overflow: hidden;
+  }
+
+  &__group {
+    display: flex;
+    align-items: center;
+  }
+
+  &__input {
+    .reset-input();
+    flex: 1;
+    // Overrides browser default. Actual width handled by flex.
+    width: 0;
+  }
+
+  &__title {
+    .link-white();
+    .link-plain();
```

Comment:
I'm starting to wonder if we should make .link-plain-white or something seeing how often this pops up